### DISCUSSION TITLE
Graph seed?

### DISCUSSION THREAD

**Discussion by christianv78** [Acly/krita-ai-diffusion#1455] [General]
I managed to figure out how to create a custom graph and import the .json file back into Krita.  But it seems like the seed number is hard-coded into the json file.  Is this the expected behavior?  Is there a way to randomize it?

**Comment by nabileisa** [comment]
use the `krita canvas` node and connect the seed output from it to the seed input in your ksampler,  this will allow you to use the same seed mechanism next to the generate button
![image](https://github.com/user-attachments/assets/81db9c1e-3c0b-4337-87fa-e766e4f32c75)
